src/workloads/execution/UnshardedGraphLookup.yml

```diff
@@ -149,6 +150,74 @@ Actors:
             {$project: {s: 0, "matches.s": 0}}
           ]
         cursor: {batchSize: *NumDocs}
+    - OperationMetricsName: GraphLookupShardedToUnshardedMaxDepthZeroOneToOne
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline:
+          [
+            {$graphLookup: {
+              from: "Collection1",
+              startWith: "$a",
+              connectFromField: "a",
+              connectToField: "b",
+              as: "matches",
+              maxDepth: 0
+            }},
+            {$project: {s: 0, "matches.s": 0}}
+          ]
+        cursor: {batchSize: *NumDocs}
+    - OperationMetricsName: GraphLookupShardedToUnshardedMaxDepthZeroOneToFive
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline:
+          [
+            {$graphLookup: {
+              from: "Collection1",
+              startWith: "$d",
+              connectFromField: "d",
+              connectToField: "e",
+              as: "matches",
+              maxDepth: 0
+            }},
+            {$project: {s: 0, "matches.s": 0}}
+          ]
+        cursor: {batchSize: *NumDocs}
+    - OperationMetricsName: GraphLookupShardedToUnshardedMaxDepthZeroOneToOneHundred
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline:
+          [
+            {$graphLookup: {
+              from: "Collection1",
+              startWith: "$f",
+              connectFromField: "f",
+              connectToField: "f",
+              as: "matches",
+              maxDepth: 0
+            }},
+            {$project: {s: 0, "matches.s": 0}}
+          ]
+        cursor: {batchSize: *NumDocs}
+    - OperationMetricsName: GraphLookupShardedToUnshardedMaxDepthZeroMatchOneToTenTargeted
```

Comment:
Could you explain what you mean by "targeted" here? My understanding is that the $graphLookup stage itself will be sent to execute on all shards in parallel, and then generate queries on "Collection 1" that will target the primary shard as it will with the other added tests. If the goal of this is to have $graphLookup execute on only a few shards I think you'll need a predicate on "c" before the $graphLookup (like there is in the MatchSomeShards test). Let me know if I'm missing something!

Reply:
Yes, this was a mistake and you're right, I really meant to be testing MatchAllShards vs MatchSomeShards. Updated in the next commit!.